app/modules/inspect/containers/configuration.jsx

```diff
@@ -0,0 +1,245 @@
+/**
+ * Copyright (c) 2014-present PlatformIO <contact@platformio.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Button, Col, Form, Row, Select, Switch } from 'antd';
+import { inspectProject, saveConfiguration } from '@inspect/actions';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { loadProjects } from '@project/actions';
+import { selectProjects } from '@project/selectors';
+import { selectSavedConfiguration } from '@inspect/selectors';
+
+class InspectionFormComponent extends React.Component {
+  static propTypes = {
```

Comment:
Group items in propTypes:
 - objects/data
- callbacks/functions, such as "loadProjects: PropTypes.func.isRequired", etc.